--- synapse/python_dependencies.py.orig	2022-04-28 23:12:38.000000000 +0200
+++ synapse/python_dependencies.py	2022-04-28 23:12:47.000000000 +0200
@@ -80,7 +80,7 @@
     # is out in November.)
     "attrs>=19.1.0",
     "netaddr>=0.7.18",
-    "Jinja2>=2.9",
+    "Jinja2<3.1.0",
     "bleach>=1.4.3",
     "typing-extensions>=3.7.4",
     # We enforce that we have a `cryptography` version that bundles an `openssl`
